human_development/global_data_lab_hid_transform_load_dlt.py

```diff
@@ -80,8 +80,9 @@ def global_data_lab_hd_index():
             'country_name',
             'adm1_name',
             'year',
```

Comment:
@weilu 

My understanding is that [global_data_lab_hid_transform_load_dlt.py](https://github.com/dime-worldbank/mega-indicators/pull/3/files#diff-f6c9a389a673e087bb1960852e585c20d532ab2e0d9746d1f9b9c1bd7eb4522e) takes "indicator_intermediate.global_data_lab_hd_index" as an input and overwrite ""indicator.global_data_lab_hd_index".

Where do we call this function?

Reply:
This is written to the target table by the DLT workflow "[Subnational Human Development Index](https://adb-6102124407836814.14.azuredatabricks.net/pipelines/a52621f7-9847-4893-aea5-d7390e29cb63?o=6102124407836814)". You can see in the settings it's pointing to the "indicator" as the target schema. The table name is as defined in the method annotation attribute in the code.